### DISCUSSION TITLE
activity.invoice_text and activity.number not showing in xls invoice

### DISCUSSION THREAD

**Discussion by triksta** [kimai/kimai#5039] [General]
### Describe the issue

Maybe i am just stupid or miss something, but i tried to adapt the invoice template "xlsx-letter" (grouped by date) and did add a field with ${activity.number} and ${activity.invoice_text} in order to get one/both to show its value. 
both fields have different values (B1 and B2 for example), but just one result is exportet (B1).
The activity.id is correctly shown.
Is it possible to get both values correctly, or did something slip through my brain?
Thanks!

### I already tried

- [X] I've read and searched [the documentation](https://www.kimai.org/documentation/).
- [X] I've searched for similar issues in this repository.
- [X] I've searched for similar issues in [the discussions](https://github.com/kimai/kimai/discussions).

### Kimai version

2.20.0

### How do you run Kimai?

Virtual Server or alike

### Which PHP version are you using?

8.1

### Screenshots

![Screenshot_20240829_202929](https://github.com/user-attachments/assets/65b2e027-ad53-4526-96af-e83baa82f35e)
![Screenshot_20240829_202930](https://github.com/user-attachments/assets/02acc407-2dcc-455c-af9d-323e87567594)
![Screenshot_20240829_203138](https://github.com/user-attachments/assets/c0970f3b-b28b-4e17-9918-ceb046bb0703)
![Screenshot_20240829_203342](https://github.com/user-attachments/assets/8486aa47-d8c0-4a88-954c-63c9db93638e)

**Comment by kevinpapst** [comment]
https://www.kimai.org/documentation/invoices.html#activity-variables

> If more than one activity was found, you will have further variables (same list as above) called ${activity.1.name}, ${activity.2.name} and so on. The order is not guaranteed, so it is not recommended relying on those variables. If your template relies on a {activity.X} variable, it is recommended to limit the invoice data with the activity search filter.

**Comment by triksta** [comment]
Thank you very much and sorry for missing this part! I will have to find a solution for my situation.
Thanks again.
